website/blog/2022-10-18-how-to-design-metrics.md

```diff
@@ -0,0 +1,367 @@
+---
+title: "How to design metrics: deciding what to model in the modeling layer vs the semantic layer"
+description: "placeholder"
```

Comment:
todo: get pithy description of this blog

Reply:
Don't love it but added something